Exercise_5.py

```diff
@@ -1,10 +1,45 @@
 # Python program for implementation of Quicksort
 
 # This function is same in both iterative and recursive
+
+# Hi TA / Jaspinder, Is there any alternative to implement iterative quicksort without using a stack?
```

Comment:
We use stack to replicate the recursion behavior. Using stack will give us a way to have a controlled recursion. I am not aware of how we can execute this task without using a stack.  @jaspindersingh83 Please comment on this if we have a different approach here to execute the same without using a stack.